poll/public/js/translations/ar/textjs.js

```diff
@@ -4,36 +4,90 @@
                   init: function() {
                     
 
-(function (globals) {
+(function(globals) {
```

Comment:
Why the spacing has been changed? I seems necessary.

Usually those files shouldn't be touched manually.

Reply:
Its not touched manually instead its generated in an automated way. I am not sure why this space came.

Reply:
Thanks @umarmughal824, I'll take a look and let you know.